File: Modul 4/Soal 5/PRAK405-2410817210010-AryaArrozzaRidhoSyaputra.c
#include <stdio.h>

int main ()
{
    int num, mul, total = 0;

    scanf("%d %d", &num, &mul);

    for (int i = 1; i <= num; i++) {
        int sum = 0;
        printf("(");

        for (int j = 1; j <= i; j++) {
            printf("%d * %d", i, mul);
            sum += j * mul;

            if (i > j) {
                printf(") + (");
            }

            else if (j > i) {
                printf(") + (");
            }
            
        }
        printf(") = %d\n", sum);
        total += sum;
    }
    
    printf ("%d\n", total);
    return 0;
}

File: Modul 4/Soal 5/PRAK405-2410817210010-AryaArrozzaRidhoSyaputra.py
num, mul = map(int, input().split())
total = 0

for i in range(1, num + 1):
    Snum = 0
    print("(", end="")

    for j in range(1, i + 1):
        print(j, "*", mul, end="")
        Snum += j * mul

        if j < i:
            print(" + ", end="")

    print(") = ", Snum)
    total += Snum

print(total)